spring-cloud-dataflow-build/spring-cloud-dataflow-build-dependencies/pom.xml

```diff
@@ -43,9 +43,152 @@
 		<junit-jupiter.version>5.9.2</junit-jupiter.version>
 		<logback.version>1.2.13</logback.version>
 		<json-path.version>2.9.0</json-path.version>
+		<tomcat.version>9.0.88</tomcat.version>
+		<rsocket.version>1.1.4</rsocket.version>
+		<netty.version>4.1.109.Final</netty.version>
+		<reactor-netty.version>1.0.44</reactor-netty.version>
 	</properties>
 	<dependencyManagement>
 		<dependencies>
+			<dependency>
```

Comment:
Please read through https://github.com/spring-cloud/spring-cloud-dataflow-release/blob/master/docs/scdf/how-to/override-boot-dependency.adoc#update-spring-cloud-dataflow-build - no need to do this manual dep mgmt as the [SB dependencies](https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/2.7.18) already pulls in the netty-bom - just update the well-known version name.